docs/PROFILES.md

```diff
@@ -0,0 +1,28 @@
+# Profiles pages
+
+## Testing
+
+### Firefox
+
+To test the `profiles` app locally on __Firefox__ :
+1. Edit `bridgeConnection.saga` on `l17` 
+```
+const extensionId = '';
```

Comment:
Why not use an .env variable ?

Reply:
see https://github.com/dis-moi/extension/pull/642#issuecomment-641491509

I'm implementing a little hack...